src/ScikitLearnAPI.jl

```diff
@@ -0,0 +1,52 @@
+
+module ScikitLearnAPI
+
+
+using Tables
```

Comment:
Curious, but I couldn't see where Tables is used in this module.

Reply:
Removed. I forgot to remove it when converting the ScikitLearn.jl code. Good catch.